pkg/apis/garden/types.go

```diff
@@ -72,6 +72,12 @@ type CloudProfileSpec struct {
 	Packet *PacketProfile
 	// CABundle is a certificate bundle which will be installed onto every host machine of the Shoot cluster.
 	CABundle *string
+	// Backup holds the object store configuration for the backups of shoot(currently only etcd).
+	// If it is not specified, then there won't be any backups taken for Shoots associated with this CloudProfile.
+	// If backup field is present in CloudProfile, then backups of the etcd from Shoot controlplane will be stored under the
+	// configured object store.
+	// +optional
```

Comment:
optional tag not needed in internal version. remove it entirely from this file.

Reply:
Done